multishard_mutation_query.cc

```diff
@@ -753,7 +760,9 @@ future<typename ResultBuilder::result_type> do_query(
         tracing::trace_state_ptr trace_state,
         db::timeout_clock::time_point timeout,
         noncopyable_function<ResultBuilder(const compact_for_query_state_v2&)> result_builder_factory) {
-    auto ctx = seastar::make_shared<read_context>(db, s, cmd, ranges, trace_state, timeout);
+    auto& table = db.local().find_column_family(s);
+    auto erm = table.get_effective_replication_map();
+    auto ctx = seastar::make_shared<read_context>(db, s, erm, cmd, ranges, trace_state, timeout);
 
```

Comment:
Note: multishard_mutation_query would never be called if storage_proxy only ever sends range queries to one tablet.

Reply:
Fixed in V2.